shared/redux/actions/core.js

```diff
@@ -11,6 +11,48 @@ const debug = (...args) => console.log(...args)
 
 const getOrders = (orders) => {
   reducers.core.getOrders({ orders })
+  actions.core.addCurrencyFromOrders(orders)
+}
+
+const addCurrencyFromOrders = (orders) => {
+  const allCurrencyies = getState().currencies.items // все валюты достпуные в клиенте
+  const partialCurrency = getState().currencies.partialItems // получаем все премиальные валюты
+  const partialArray = partialCurrency.map(item => item.name) // получаем их имена
+
+  const sellOrderArray = orders.map(item => item.sellCurrency) // получаем из ордерова валюты на продажу
+  const buyOrderArray = orders.map(item => item.buyCurrency) // получаем из ордерова валюты на покупку
+
+  let sortedArray = [...sellOrderArray] // записываем sellOrderArray в массив
+
+  for (let i = 0; i <= sellOrderArray.length - 1; i++) {
```

Comment:
```suggestion
  for (const i of sellOrderArray) {
```

можно хотя бы так)

![image](https://user-images.githubusercontent.com/1909384/53703298-848a2680-3e21-11e9-84b0-91bf16bb34a3.png)